Applications/TransferLearning/Draw_Classification/README.md

```diff
@@ -55,40 +69,31 @@ Input_Shape = 1:1:128         # Input shape (without batch size)
 [fc1layer]                    # Layer Name
 Type = fully_connected        # Layer Type
 Unit = 20                     # output unit
-Bias_init_zero = true         # bias initialization
+Bias_initializer = zeros      # bias initialized to 0
 Activation = sigmoid          # activation
 
 [outputlayer]                 # Layer Name
 Type = fully_connected        # Layer Type
 Unit = 3                      # output unit
-Bias_init_zero = true         # bias initialization
+Bias_initializer = zeros      # bias initialized to 0
 Activation = softmax          # activation
 ```
 
 ### How to run
 
-It takes the input image and ```ssd_mobile_v2_coco_feature.tflite``` is used to extract features.
-Feature data is feeded into network described earlier, and it is uesd to train network.
-
-We can run this as below
+Once the application has been build with meson, use the instructions below to run:
 
 ```bash
 $ pwd
   build
-$ cd ..
-$ build/Applications/TransferLearning/Draw_Classification/jni/nntrainer_training Applications/TransferLearning/Draw_Classification/res/Trainig.ini Applications/TransferLearning/Draw_Classification/res/
+$ ./Applications/TransferLearning/Draw_Classification/jni/nntrainer_training ../Applications/TransferLearning/Draw_Classification/res/Training.ini ../Applications/TransferLearning/Draw_Classification/res/
 
 ```
 
 ### Results
 
-After Iterating 300 times, the change of L2 Norm of the Loss function is below.
-<p align = "center">
-<img src="https://github.com/nnstreamer/nntrainer/blob/master/docs/images/d42b1300-f0cf-11e9-9b6f-6db30def4684.png" width="500" height="300">
-</p>
-
-and the test results for the 8 test cases are below. Step function is used to make more clear.
-As you can see, the test result is ok.
+The training reduces the training loss starting from `1.08` to `0.0048`.
+The test results for the 8 test cases are below. Top-1 prediction is used to check the results.
 
 <p align ="center">
```

Comment:
Let's merge this for now and make another pr for the update which includes nnstreamer for ubuntu and result.